src/SWP/Component/Common/Model/TimestampableTrait.php

```diff
@@ -26,6 +26,11 @@ trait TimestampableTrait
      */
     protected $updatedAt;
 
+    public function __construct()
+    {
+        $this->createdAt = new \DateTime();
```

Comment:
it will be called also when class using it will have his own constructor?

Reply:
when that Trait will be used by the class having its own constructor, the `createdAt` has to be set manually in that class.

When there is no constructor defined in the class, the value of `createdAt` will be set based on the Trait constructor.